src/Chapter4.hs

```diff
@@ -628,8 +661,14 @@ Can you implement a monad version of AND, polymorphic over any monad?
 
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
+andMv1 :: (Monad m) => m Bool -> m Bool -> m Bool
+andMv1 a b = fmap (&&) a <*> b
+
 andM :: (Monad m) => m Bool -> m Bool -> m Bool
-andM = error "andM: Not implemented!"
+andM a b = a >>= \x -> pure (x &&) <*> b
```

Comment:
This version is similar to the previous one. Because `pure f <*> x` is the same as `fmap f x`